lib/globals.d.ts

```diff
@@ -1 +1,11 @@
 declare var browser: any;
+
+declare class WebDriver {
```

Comment:
I'm in favor of everything that's required from WebDriver being put in a namespace.